core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFormatterUtil.java

```diff
@@ -61,7 +66,7 @@ interface DateTimeFormatHandler {
       .put("%D", (date) -> // %w - Day of month with English suffix
           String.format("'%d%s'", date.getDayOfMonth(), getSuffix(date.getDayOfMonth())))
       .put("%f", (date) -> // %f - Microseconds
-          String.format("'%d'", (date.getNano() / 1000)))
+          String.format(NANO_SEC_FORMAT, (date.getNano() / 1000)))
```

Comment:
Love it - but I feel like we should use a constant for all of the surrounding format strings.